### ISSUE TITLE
cleanup-db job fail

### ISSUE THREAD

**Issue by lionelB** [SocialGouv/cdtn-admin#435]
cleanup-db fail

```
  2021-05-01 06:12:27  ERROR:  invalid input syntax for type timestamp with time zone: "3 month" 
  2021-05-01 06:12:27  LINE 5: DELETE FROM alerts WHERE created_at < NOW() - '3 month' AND ...
  2021-05-01 06:12:27                                                        ^
```

**Comment by douglasduteil** [comment]
@revolunet You know from where this is ?
@lionelB can you provide a link to the logs plz ?

**Comment by lionelB** [comment]
>  @lionelB can you provide a link to the logs plz ?

see in grafana but I don't have the request

**Comment by lionelB** [comment]
fixed in  #450
